test/unit/lifecycle/properties.js

```diff
@@ -464,9 +465,58 @@ describe('lifecycle/property', function () {
         elem.test = 'something else';
         expect(called).to.equal(1);
       });
+
+      it('context and arguments', function (done) {
+        const opts = {
+          created (elem, data) {
+            expect(this.created).to.equal(opts.created);
+            expect(arguments.length).to.equal(2);
+            expect(elem.tagName).to.equal('DIV');
+            expect(data.name).to.equal('test');
+            expect(data.internalValue).to.equal(undefined);
+            done();
+          }
+        };
+        create(opts);
+      });
+    });
+
+    describe('initial', function () {
```

Comment:
Is there a test where the `initial` callback returns a value that is then used as the internal value? I think that would be useful to have